packages/yourloops/pages/hcp/patients-list.tsx

```diff
@@ -274,7 +276,7 @@ class PatientListPage extends React.Component<RouteComponentProps, PatientListPa
     } else if (filterType === "pending") {
       patients = []; // TODO
     } else if (filterType !== "all") {
-      // TODO
+      patients = allPatients.filter((patient: User): boolean => patient.teams?.includes(filterType) ?? false);
```

Comment:
```suggestion
      patients = allPatients.filter((patient: User): boolean => patient.teams?.includes(filterType));
```

Reply:
My next PR will change this code

Reply:
Fix in https://github.com/mdblp/blip/pull/400